candi/bashible/bashible.sh.tpl

```diff
@@ -251,6 +252,9 @@ function main() {
       echo ===
       echo === Step: $step
       echo ===
+      {{- if eq .runType "ClusterBootstrap" }}
+      if [ "$attempt" -gt 2 ];then sx=x;fi
```

Comment:
```suggestion
      if [ "$attempt" -gt 2 ]; then 
        sx="x"; 
      fi
```

Reply:
More readable

Reply:
I tried to economy some bytes for aws, but yes this code will render for bootstrap first master node only. Thank you, I've fixed it